### ISSUE TITLE
Allow polytopes in real projective space

### ISSUE THREAD

**Issue by ChristopherKing42** [vihart/hypernom#1]
SO(3) is topologically the same as real projective space. What if there was a mode where you used this space instead of the 3-sphere? Then you would only need to go through every position once, not twice.

In addition, real projective space is non-orientable. I bet there are some cool ways this property could be used (making things different colors when they swap orientation, for example).

**Comment by ChristopherKing42** [comment]
Indeed, it appears that any [spherical 3-manifold](https://en.wikipedia.org/wiki/Spherical_3-manifold) would work.

**Comment by henryseg** [comment]
The user would still see in S^3 rather than RP^3 - I think that the code would only change in that both antipodal cells would vanish when you touch either one. Perhaps rather than a colouring change, it would make sense for the cell to be replaced with some geometry that has a handedness - perhaps a spiral is drawn on each cell-face. Then you can see which handedness you have relative to the spiral. Although there is no consistent way to choose the handedness of each cell in RP^3, so I'm not sure how one would do it.

**Comment by ChristopherKing42** [comment]
The idea of handiness does make sense in RP^3, but only locally, instead of globally. For instance, in RP^2, you can put a spiral on each face of a [Hemi-octahedron](https://en.wikipedia.org/wiki/Hemi-octahedron), such that the spiral goes in the opposite of each of its neighbors. There is probably something similar you can do in RP^3.
